routing/router.go

```diff
@@ -1,43 +1,72 @@
 package routing
 
 import (
+
 	"net/http"
 
 	"github.com/gorilla/mux"
 
 	"github.com/kiali/kiali/config"
+
+	"github.com/kiali/kiali/log"
```

Comment:
trivial - but get rid of this blank line above the kiali/log import. The kiali imports should be one block so gofmt can sort them all. I would also get rid of the blank line on line 4.